packages/syft/src/syft/service/user/user.py

```diff
@@ -144,10 +177,26 @@ class UserUpdateV1(PartialSyftObject):
 
 
 @serializable()
-class UserUpdate(PartialSyftObject):
+class UserUpdateV2(PartialSyftObject):
```

Comment:
Just resolved conflict with dev but doesn't seem to be any difference between `UserUpdate` and `UserUpdateV2`?

Reply:
Ohh yeah, in fact there's no difference between `UserUpdate/UserUpdateV2`. Also, after you pointed it, I noticed that `UserCreate/UserCreateV2`  didn't change either. So I pushed a [new commit](https://github.com/OpenMined/PySyft/pull/8455/commits/064e2768270f7341cd8512b982d5a51085f8ae80) rolling back both.

Thank you for noticing it @kiendang ! ❤️